### DISCUSSION TITLE
Purchased services emission boundary

### DISCUSSION THREAD

**Discussion by arby1005** [USEPA/supply-chain-factors#33] [Q&A] [answered]
Hi all,

When calculating purchased services' emissions, I wonder if the EEIO emission factor covers the service providers' Scope 1 and 2 emissions, and their upstream emissions (i.e., their Scope 3 cat 1 emissions etc.). 

For example, what are their emission boundaries for advertising, marketing or consultation services purchased?

Thank you

**Comment by WesIngwersen** [comment, marked as answer]
hi @arby1005.  Yes, they would be included, at least to the extent that our Supply Chain Factors cover Scope 3 (see webinar linked from this repo main page for more on that).
